app/_locales/en/messages.json

```diff
@@ -885,9 +885,15 @@
   "confirmAlertModalDetails": {
     "message": "If you sign in, a third party known for scams might take all your assets. Please review the alerts before you proceed."
   },
+  "confirmAlertModalMessageDomainMismatch": {
+    "message": "The site making the request is not the site you’re signing into. This could be an attempt to steal your login credentials."
+  },
   "confirmAlertModalTitle": {
     "message": "Your assets may be at risk"
   },
+  "confirmAlertModalTitleSignIn": {
```

Comment:
For comparison, I've been going with `alertReasonX` and `alertMessageX` for the transaction alerts.

My thinking is it stops it being as long, plus decouples the alerts from confirm which doesn't really add any extra info.

Reply:
I was thinking this and was concerned about diverting from the original patterns set in place. Sounds like a good update to me

Reply:
updated 07b8a97 & abc8365